tests/evaluate/logging/utils.py

```diff
@@ -0,0 +1,230 @@
+from typing import List
+
+import pandas as pd
+
+from mlflow.entities.evaluation import Evaluation as EvaluationEntity
+from mlflow.evaluation.utils import (
+    _get_assessments_dataframe_schema,
+    _get_evaluations_dataframe_schema,
+    _get_metrics_dataframe_schema,
+    _get_tags_dataframe_schema,
+)
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, RESOURCE_DOES_NOT_EXIST
+from mlflow.tracking.client import MlflowClient
+
+
+def get_evaluation(*, run_id: str, evaluation_id: str) -> EvaluationEntity:
```

Comment:
This is just a test utility to verify that evaluations are logged properly. In the future, when we support read APIs in the MLflow client, we will move the test utility into the MLflow fluent / client APIs.